### DISCUSSION TITLE
something about genome reference

### DISCUSSION THREAD

**Discussion by jefferyUstc** [stuart-lab/signac#499] [Q&A] [answered]
Hi， I got a question about `BSgenome.Hsapiens.UCSC.hg38` and `EnsDb.Hsapiens.v86`, from internet, i get to know that both two are annotations of hg38, one is UCSC hg38, another is ensemble hg38. my peaks were called by ucsc hg38. So， what if i run with below codes:
```
annotations <- GetGRangesFromEnsDb(ensdb = EnsDb.Hsapiens.v86)
seqlevelsStyle(annotations) <- 'UCSC'
genome(annotations) <- "hg38"
Annotation(seurat_obj) <- annotations
```

is that inappropriate ?

**Comment by timoast** [comment, marked as answer]
If your data was mapped to hg38 (UCSC-style genome), then you need to supply gene annotations in the same style, so it makes sense to change the `seqlevelStyle` here to UCSC for the gene annotations.
